### PR TITLE
[chore] update components with no active codeowners to unmaintained

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

**Review by songy23 — APPROVED** [review]

**Review by codeboten — APPROVED** [review]
needs a rebase

**Comment by songy23** [comment]
When you rebase, could you also fix the blank line in `.github/CODEOWNERS` https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/11482950662/job/31957336727  🙏

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -322,3 +320,6 @@ reports/distributions/contrib.yaml @open-telemetry/collector-contrib-approvers
 
 
 ## UNMAINTAINED components
+
+extension/observer/ecstaskobserver/                                 @open-telemetry/collector-contrib-approvers
```
Comment (songy23) [inline .github/CODEOWNERS:324]:
```suggestion
extension/observer/ecstaskobserver/                                 @open-telemetry/collector-contrib-approvers 
```

**Comment by atoulme** [comment]
Rebased.

**Review by codeboten — APPROVED** [review]

**Pull request merged**